### DISCUSSION TITLE
Domain's DNS record could not be retrieved.

### DISCUSSION THREAD

**Discussion by Ussef-khalfi** [community/community#49393] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, please if someone can help me i put a custom domain on my repository but when i check it, it appear this message: Both youssefelkhalfi.com and its alternate name are improperly configured
Domain's DNS record could not be retrieved. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidDNSError).
if can someone help thanks.

**Comment by airtower-luna** [comment]
That domain doesn't seem to exist at all. If you want to use a custom domain with Pages you have to register it first, and then point it at the Pages servers (as described in the linked documentation).
